os/dbuild.sh

```diff
@@ -105,9 +105,9 @@ function SELECT_OPTION()
 			echo ======================================================
 			echo "  \"1. Build with Current Configurations\""
 			echo "  \"2. Re-configure\""
-			echo "  \"3. Menuconfig\""
-			echo "  \"4. Build Clean\""
-			echo "  \"5. Build Dist-Clean\""
+			echo "  \"3. Modify Current Configuration\""
+			echo "  \"4. Clean Build\""
+			echo "  \"5. Clean Build with New Configuration\""
```

Comment:
could you reword it to something else? Like **Clean Build and Clear Current Configuration** or something else? I feel it's better to inform the user that we are removing the current configuration.

Reply:
I also think @abhishek-samsung 's suggestion is better to know what it is.
And minor one.
`configurations` or `configuration`? Could you use the same at all of the list?
`1. build  with current configurations` has `s`.

Reply:
Yes Clean Build and Re-configuration is more precise.
I have modified all to configuration.